localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,186 @@
+ 
+ innovation_naga_taming:0 "Naga Taming"
+ innovation_naga_taming_desc:0 "We have tamed the Couatl and enslaved the Sea Giants, and now we have encorporated them into our army."
+ 
+ innovation_savage:0 "Savages"
+ innovation_savage_desc:0 "Savages are fanatic and barbaric warriors of our clan, they are strong and stubborn, and show no mercy."
+ 
+ innovation_silithid:0 "Silithid"
+ innovation_silithid_desc:0 "Silithids are insectoid creatures that are very strong and sophisticated. The chitin plates of their bone armor are intertwined in such a way that it makes it incredibly hard. It seems that the Silithids' intention is to devour everything that stands in their way and that does not belong to their own ecology. This is the only thing they focus on, and this is why they pose a great threat to all life on Azeroth."
+
+ innovation_snow_hunter:0 "Snow Hunters"
+ innovation_snow_hunter_desc:0 "Our kind has had to adapt to the elements in order to survive.  The best of our warriors are the hunters who keep us safe."
+    
+ innovation_thornguard:0 "Thornguards"
+ innovation_thornguard_desc:0 "The Thornguards are the finest warriors from our tribe, who defend us from any outsiders.  They are heavily armed and armoredt They will stop any who dare to trespass or any who dare to not submit."
+
+ innovation_tidehunter:0 "Tidehunters"
+ innovation_tidehunter_desc:0 "mrglmrglmrglmrgl"
+ 
+ innovation_transcendent:0 "Transcendent Avatars"
+ innovation_transcendent_desc:0 "Pandaren revere transcendents as embodiments of wisdom: the spirit of the universe made flesh, residing in the shell of a pandaren host. These are the pandaren’s ultimate heroes: Spiritual avatars capable of summoning vast storms of lighting, smashing mountains with their terrible fists, and breathing winds into existence."
+    
+ innovation_ursa:0 "Ursas"
+ innovation_ursa_desc:0 "Ursas are the biggest, meanest and most feral of furbolgs. They tower above humans and orcs, and can snap elven necks with single paw swipes. They are immensely strong and tough, and focus on improving their natural, bearlike natures. They disembowel dragons with their claws and crush small creatures that come within their grasp."
+
+ innovation_war_dinosaur:0 "War Dinsosaurs"
+ innovation_war_dinosaur_desc:0 "Once a dominant species throughout the planet, they have become an endangered remnant of an ancient world that prove to be a force to be reckoned with. These great beasts are as dangerous as they are majestic. Our tamers have successfully learned how to communicate and tame these grand creatures."
+
+ innovation_wolfrider:0 "Wolfriders"
+ innovation_wolfrider_desc:0 "These savage warriors use the speed and ferocity of their mounts to make them a deadly enemy. Faster than those who attack on foot, they have perfected the tactics of tearing through columns of enemy soldiers like a hot wind."
```

Replaced savage since it's another MaA names:
```suggestion
 innovation_wolfrider_desc:0 "These fierce warriors use the speed and ferocity of their mounts to make them a deadly enemy. Faster than those who attack on foot, they have perfected the tactics of tearing through columns of enemy soldiers like a hot wind."
```